torch_xla/csrc/runtime/topology.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package xla.tpu;
+
+option cc_enable_arenas = true;
+
+// Describes features of a tpu.
+message TPUHardwareFeature {
+  // Embedding feature of a tpu.
+  enum EmbeddingFeature {
+    // No embedding lookup accelerator available on the tpu.
+    UNSUPPORTED = 0;
+    // Embedding lookup accelerator V1. The embedding lookup operation can only
+    // be placed at the beginning of computation. Only one instance of embedding
+    // lookup layer is allowed.
+    V1 = 1;
+    // Embedding lookup accelerator V2. The embedding lookup operation can be
+    // placed anywhere of the computation. Multiple instances of embedding
+    // lookup layer is allowed.
+    V2 = 2;
+  }
+  EmbeddingFeature embedding_feature = 1;
+}
+
+// Describes the geometry of a TPU mesh.
+message TopologyProto {
```

Comment:
why do we need this? We should be able to remove mesh service after XRT. PJRT does not use mesh service.

Reply:
PJRT has a test for mesh service: https://github.com/pytorch/xla/blob/master/test/pjrt/test_mesh_service.py

not sure whether this test is still needed or not

Reply:
mesh service would been deleted in https://github.com/pytorch/xla/pull/5240#pullrequestreview-1495708204, will delete `topology.proto` after rebase with merged #5240

Reply:
has rebased to Jun27, topology has been deleted before rebase